### ISSUE TITLE
Feature Request: Monitor folder, automatically import data

### ISSUE THREAD

**Issue by andres-GHub** [veusz/veusz#200]
hi,I currently have a few different hundred folders of categories which are being populated every so often with time-stamped folders that have in them the same .txt data files. At the moment, I import each txt file manually into veusz along with the appropriate corresponding user defined Prefix and Tag. 
Would it be possible to implement a feature were a particular folder is being monitored (every so often) so whenever a new folder is added, veusz automatically will go into this folder and search for a specific .txt file which it then loads into veusz with a user defined Prefix, Tag, etc.?
thanks in advance

**Comment by mianzhi** [comment]
I'd prefer keeping the current Veusz lean and simple.
For the requested task, I would:
1. create a template .vsz file, replace the .txt file path with an identifier string.
2. write a script to clone the template and override the identifier string for each matching .txt data.

**Comment by jeremysanders** [comment]
Yes - I think this request is too much to include in the standard Veusz.

However, it might be worth extending the ToolsPlugin interface to allow a plugin to be called at intervals. Then a user could relatively easily add code to do this from within the program.
